pyproject.toml

```
@@ -0,0 +1,30 @@
+[tool.poetry]
+name = "epcsunspecdemo"
+version = "0.1.0.post46.dev0+2a57176"
```

The `poetry-dynamic-versioning` docs suggests:

> Poetry's typical version setting is still required in `[tool.poetry]`, but you are encouraged to use `version = "0.0.0"` as a standard placeholder.

```suggestion
version = "0.0.0"
```